File: src/Uri/_2_August/URI_1070_SixOddNum.java
package Uri._2_August;

import java.util.Scanner;

public class URI_1070_SixOddNum {
    public static void main(String[] args) {
        int x;
        Scanner sc = new Scanner(System.in);
        x = sc.nextInt();

        if (x % 2 == 0) {
            x += 1;
        }
        for (int i = 0; i <= 5; i++) {
            if (x % 2 != 0) {
                System.out.println(x);
            }
            x += 2;
        }
    }
}

File: src/Uri/_2_August/URI_1070_SixOddNum.kt
package Uri._2_August

import java.util.*

fun main(args: Array<String>) {
    var num: Int;
    var sc = Scanner(System.`in`);
    num = sc.nextInt();

    if (num % 2 == 0) {
        num += 1;
    }

    for (i in 0..5) {
        if (num % 2 != 0) {
            println(num)
        }
        num+=2;
    }
}